src/product-info.js

```diff
@@ -32,6 +32,10 @@ export default {
     gaStatus: 'GA'
   },
   rhsso: {
+    prettyName: 'Red Hat Single Sign-On (Cluster)',
```

Comment:
@tiffanynolan this doesn't align with doc at https://integr8ly.github.io/user-documentation/index.html?e=docs/getting-started.adoc#gs-adding-users-proc

but we can adjust as required

Reply:
@finp I like how you referred to the 2 instances in the docs as those terms sound more user friendly. Ultimately, I'd like to only show the end-user SSO in the applications list and move the cluster level one to more of an admin screen/settings page. We're not there yet though so we do need some way to easily tell them apart for now.

Reply:
@tiffanynolan @finp Just to confirm, should I be changing the naming of this in the web app (and if so, what to)? Or will it be changed in the documentation?

Reply:
@aidenkeating my reading of  @tiffanynolan comment is to go with what we have. I could add a note to docs if required.